control-plane-operator/hostedclusterconfigoperator/controllers/resources/ingress/reconcile.go

```diff
@@ -80,6 +81,24 @@ func ReconcileDefaultIngressController(ingressController *operatorv1.IngressCont
 				},
 			},
 		}
+	case hyperv1.OpenStackPlatform:
+		ingressController.Spec.EndpointPublishingStrategy = &operatorv1.EndpointPublishingStrategy{
+			Type: operatorv1.LoadBalancerServiceStrategyType,
+			LoadBalancer: &operatorv1.LoadBalancerStrategy{
+				Scope: loadBalancerScope,
+				ProviderParameters: &operatorv1.ProviderLoadBalancerParameters{
+					Type: operatorv1.OpenStackLoadBalancerProvider,
+					// TODO(emilien): add the field once bumped openshift/api and also remove `ReconcileDefaultIngressControllerWithUnstructured`.
```

Comment:
Does this need uncommented?

Reply:
I suggest we leave it like this, I made it a separate commit so we can revert it once we bump the openshift/api here.

Reply:
I left it here, purely for "documentation" that what we in-fine expect to be the code.

Reply:
I'm happy to remove the whole commented block.